portal-api/src/scheduler/jobs/helpers/utilisation-report-helpers.js

```diff
@@ -78,7 +79,10 @@ const getFormattedReportPeriod = () => {
  */
 const getIsReportSubmitted = async (bank) => {
   const reportPeriod = getReportPeriod();
-  const reportsResponse = await api.getUtilisationReports(bank.id, { reportPeriod });
+  const nonNotReceivedStatuses = Object.values(UTILISATION_REPORT_RECONCILIATION_STATUS).filter(
+    (status) => status !== UTILISATION_REPORT_RECONCILIATION_STATUS.REPORT_NOT_RECEIVED,
+  );
```

Comment:
We've needed this set of statuses quite a few times now, so probably makes sense to add a `getUtilisationReportReceivedStatuses` helper, maybe in the same place as the `UTILISATION_REPORT_RECONCILIATION_STATUS` constant.

Reply:
I was hoping to add that as a helper in `common` which I want to do in a separate PR after this branch is merged (I think it would be nice to have a commit at the end of this refactor that commonises a lot of the new things we've added)

Reply:
Also, all statuses which are not `REPORT_NOT_RECEIVED` do not necessarily correspond to a report being received, hence the name `nonNotReceivedStatuses` (unless we decide that 'Received' and 'Uploaded' mean different things)

Reply:
Yeah, I'd thought about that, but then to me `nonNotReceivedStatuses` means the same thing 🤷‍♂️

Reply:
The other way I can think of writing this to make it clear is to make an `isReportUploaded` filter like
```typescript
const isReportUploaded = (report: UtilisationReport) =>
  report.status !== 'REPORT_NOT_RECEIVED' && !!report.azureFileInfo;
```
and to use it as follows:
```typescript
const uploadedReports = (
  await api.getUtilisationReports(bank.id, { reportPeriod }),
).filter(isReportUploaded);
```
What do you think?

Reply:
Yeah, that looks good to me. Much clearer what the criteria is and more concise. Nice 👍